cost-analyzer/templates/NOTES.txt

```diff
@@ -1,44 +1,20 @@
-
-
 --------------------------------------------------
-{{- include "kubecostV2-preconditions" . }}
-
-{{- if (.Values.podSecurityPolicy).enabled }}
-{{- fail "Kubecost no longer includes PodSecurityPolicy by default. Please take steps to preserve your existing PSPs before attempting the installation/upgrade again with the podSecurityPolicy values removed." }}
-{{- end }}
-
-{{- $isEKS := (regexMatch ".*eks.*" (.Capabilities.KubeVersion | quote) )}}
-{{- $isGT22 := (semverCompare ">=1.23-0" .Capabilities.KubeVersion.GitVersion) }}
-{{- $PVNotExists := (empty (lookup "v1" "PersistentVolume" "" "")) }}
-{{- $EBSCSINotExists := (empty (lookup "apps/v1" "Deployment" "kube-system" "ebs-csi-controller")) }}
-
-{{- $servicePort := .Values.service.port | default 9090 }}
+{{- include "kubecostV2-preconditions" . -}}
+{{- include "eksCheck" . -}}
+{{- $servicePort := .Values.service.port | default 9090 -}}
 Kubecost {{ .Chart.Version }} has been successfully installed.
 
-{{ if (and $isEKS $isGT22) -}}
-
-WARNING: ON EKS v1.23+ INSTALLATION OF EBS-CSI DRIVER IS REQUIRED TO MANAGE PERSISTENT VOLUMES. LEARN MORE HERE: https://docs.kubecost.com/install-and-configure/install/provider-installations/aws-eks-cost-monitoring#prerequisites
-
-{{ if (and $EBSCSINotExists $PVNotExists) -}}
+Welcome to Kubecost 2.0!
 
-ERROR: MISSING EBS-CSI DRIVER WHICH IS REQUIRED ON EKS v1.23+ TO MANAGE PERSISTENT VOLUMES. LEARN MORE HERE: https://docs.kubecost.com/install-and-configure/install/provider-installations/aws-eks-cost-monitoring#prerequisites
-
-{{ else if (and $EBSCSINotExists (not $PVNotExists)) -}}
-
-ERROR: MISSING EBS-CSI DRIVER WHICH IS REQUIRED ON EKS v1.23+ TO MANAGE PERSISTENT VOLUMES. LEARN MORE HERE: https://docs.kubecost.com/install-and-configure/install/provider-installations/aws-eks-cost-monitoring#prerequisites
-
-{{ end -}}
-{{ end -}}
-
-
-Please allow 5-10 minutes for Kubecost to gather metrics.
-
-When configured, cost reconciliation with cloud provider billing data will have a 48 hour delay.
+Kubecost 2.0 is a major upgrade from previous versions and includes major new features including a brand new API Backend. Please review the following documentation to ensure a smooth transition: https://docs.kubecost.com/v2.0/
+For the full list of enhancements, please see our release notes: https://github.com/kubecost/cost-analyzer-helm-chart/releases/tag/v2.0.0
 
 When pods are Ready, you can enable port-forwarding with the following command:
 
-    kubectl port-forward --namespace {{ .Release.Namespace }} deployment/{{ template "cost-analyzer.fullname" . }} {{ $servicePort }}
+    kubectl port-forward --namespace kubecost deployment/kubecost-cost-analyzer 9090
```

Comment:
Why was this changed to hard-code values?

Reply:
good catch!!! copy paste error